websites/P/Phind/presence.ts

```diff
@@ -0,0 +1,72 @@
+const presence = new Presence({
+		clientId: "1106990410838065172",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "phind-square",
+			startTimestamp: browsingTimestamp,
+			details: "Browsing Phind",
+		},
+		pathDetailsMap = {
+			"/default": "Viewing how to set Phind as default",
+			"/bangs": "Viewing !Bangs",
+			"/mobile": "Viewing Mobile Page",
+			"/hotkeys": "Viewing Hotkeys",
+			"/history": "Viewing Search History",
+			"/about": "Viewing About Page",
+			"/tutorial": "Viewing Tutorial",
+			"/privacy": "Viewing Privacy Policy",
+			"/terms": "Viewing Terms",
+		},
+		pathDetails = Object.entries(pathDetailsMap).find(([pathPrefix]) =>
+			document.location.pathname.startsWith(pathPrefix)
+		);
+
+	if (pathDetails) {
+		presenceData.details = pathDetails[1];
+		presenceData.state = document.location.pathname;
+	} else {
+		const searchResults = document.querySelectorAll("div.row[name^='answer-']");
+		if (searchResults.length > 0) {
+			const searchDetails = searchResults[
+				searchResults.length - 1
+			].querySelector("span.fw-bold.fs-3.mb-3").textContent;
+
+			presenceData.details = "Searching...";
+			presenceData.state = searchDetails;
+			presenceData.smallImageKey = "search";
+		}
+	}
+
+	const settings = {
+		displaySearch: await presence.getSetting("displaySearch"),
+		shareSearch: await presence.getSetting("shareSearch"),
+		privateMode: await presence.getSetting("privateMode"),
+	};
```

Comment:
Use `await Promise.all(...)` to load multiple settings.

Reply:
Changed to
```typescript
const [displaySearch, shareSearch, privateMode] = await Promise.all([
	presence.getSetting("displaySearch"),
	presence.getSetting("shareSearch"),
	presence.getSetting("privateMode"),
]);
```